apps/api/src/report/report.util.service.ts

```diff
@@ -215,6 +215,120 @@ export class ReportUtilService {
       FROM report WHERE registration_date::date >= '${from}';
     `;
   }
+  /**
+   *
+   * @param mappedStatusesString
+   * @param from start
+   * @param to end
+   * @param BCCNM_NEW_PROCESS
+   * @returns Query string
+   */
+  reportFour(
+    mappedStatusesString: string,
+    from: string,
+    to: string,
+    BCCNM_NEW_PROCESS: boolean,
+  ): string {
+    return `
+    SELECT 
+      "status_audit"."status_id", count(status_audit.status_id)
+    FROM 
+      "ien_applicant_status_audit" "status_audit" 
+    LEFT JOIN 
+      ien_applicants applicant ON applicant.id = status_audit.applicant_id 
+    WHERE 
+      applicant.new_bccnm_process IS NOT NULL AND applicant.new_bccnm_process = ${
+        BCCNM_NEW_PROCESS ? 'TRUE' : 'FALSE'
+      } 
+      AND 
+        "status_audit"."status_id" IN 
```

Comment:
Instead of passing the map of status IDs, how about this?
```

LEFT JOIN ien_applicant_status ias ON ias.id = status_audit.status_id
WHERE
  applicant.new_bccnm_process = FALSE
  AND
    ias.category = 'IEN Licensing/Registration Process' 
```